app/scripts/modules/oracle/pipeline/stages/resizeAsg/resizeAsgExecutionDetails.controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+import {
+  EXECUTION_DETAILS_SECTION_SERVICE
+} from '@spinnaker/core';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.oraclebmcs.pipeline.stage.resizeAsg.executionDetails.controller', [
+  EXECUTION_DETAILS_SECTION_SERVICE,
+  require('angular-ui-router').default,
+  require('core/delivery/details/executionDetailsSectionNav.directive.js'),
+])
+  .controller('oraclebmcsResizeAsgExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService) {
+
+    $scope.configSections = ['resizeServerGroupConfig', 'taskStatus'];
+
+    initialize();
+
+    let initialized = () => {
+      $scope.detailsSection = $stateParams.details;
+    };
+
+    let initialize = () => executionDetailsSectionService.synchronizeSection($scope.configSections, initialized);
+
+    $scope.$on('$stateChangeSuccess', initialize);
+  });
```

Comment:
For boilerplate execution details, you can use the `BaseExecutionDetailsCtrl` instead of creating your own here (this is a fairly new thing as of nine days ago) - take a look at the other `resizeAsgExecutionDetails.html` files for reference.

The only difference is you'd include the sections in the stage registration config (I'll note below where that should go).